spec/requests/fixed_streams_spec.rb

```diff
@@ -58,6 +58,41 @@
       expect(response.status).to eq(200)
       expect(JSON.parse(response.body)).to eq(expected_response.as_json)
     end
+
+    it 'returns 1h update frequency for US EPA AirNow user' do
```

Comment:
I wonder how we should test this change 🤔 
My thoughts:
There are two possible cases for update frequency value - we can get `1 hour` or `1 minute`. I would say we should test both of them. That's why I would add a unit test for the serializer and check it there. This spec I would leave as it was.
WDYT?